src/replica/backup/replica_backup_server.h

```diff
@@ -20,13 +20,13 @@
 #include <dsn/dist/replication/replication_types.h>
 #include <dsn/cpp/rpc_holder.h>
 
+#include "common/replication_common.h"
```

Comment:
Why did you move backup_rpc to `replication_common.h`

Reply:
Seems that most rpc_hoder defined in `replication_common.h` or `replica_stub.h`.